spec/features/navigation_spec.rb

```diff
@@ -33,5 +33,79 @@
       end
 
     end
+
+    it "I cannot access any restricted routes directly" do
+      visit '/merchant/dashboard'
+      expect(page).to have_content("The page you were looking for doesn't exist (404)")
+
+      visit '/admin/dashboard'
+      expect(page).to have_content("The page you were looking for doesn't exist (404)")
+
+      visit '/profile'
+      expect(page).to have_content("The page you were looking for doesn't exist (404)")
+    end
+  end
+
+  describe "As a default user" do
+    it "I cannot access any restriced routes" do
+      user = User.create!(name: "Steve",
+                          address:"123 Main St.",
+                          city: "Fort Collins",
+                          state: "GA",
+                          zip: "66666",
+                          email: "chunky_lover@example.com",
+                          password: "123password")
+
+      visit '/login'
+      fill_in :Email, with: user.email
+      fill_in :Password, with: user.password
+      click_button "Login"
+      visit '/merchant/dashboard'
+      expect(page).to have_content("The page you were looking for doesn't exist (404)")
+      visit '/admin/dashboard'
+      expect(page).to have_content("The page you were looking for doesn't exist (404)")
+    end
+  end
+
+  describe "As a merchant" do
```

Comment:
we were also wondering if these tests should live in the navigation spec or if they should be in a more focused spec file, like navigation_restriction spec

Reply:
We will probably want to move this into a separate test, added to to-do list.